gpflow/ci_utils.py

```diff
@@ -35,12 +35,20 @@ def is_continuous_integration() -> bool:
     return "CI" in os.environ
 
 
-def ci_niter(n: int, test_n: int = 2) -> int:
+def ci_n(n: int, test_n: int = 2) -> int:
     return test_n if is_continuous_integration() else n
 
 
+def ci_npoints(n: int, test_n: int = 2) -> int:
+    return ci_n(n, test_n)
+
+
+def ci_niter(n: int, test_n: int = 2) -> int:
+    return ci_n(n, test_n)
```

Comment:
Is there a good reason to have these three identical fns? 
I'd also vote for a more descriptive name for these like "adapt_for_tests()" or  "reduce_for_tests()"